snippets/pod-volume-restore-snapshot-read-only.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * openshift-docs/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+// * openshift-docs/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-applications-restic-doc.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+.PodVolumeRestore fails with the error `.../.snapshot: read-only file system`
+====
+The `.../.snapshot` directory is a snapshot copy directory, which is used by several NFS servers. This directory has read-only access by default, so velero cannot restore to this directory.
+
+You should not give write access to this `snapshot` directory to velero, and you should disable client access to this directory.
```

Comment:
```suggestion
You should not give write access to this `.snapshot` directory to velero, and you should disable client access to this directory.
```

It's not possible to add ref to other resources like below right?

> NetApp ontap: deselect Show the Snapshot copies directory to clients or Allow clients to access Snapshot copies directory. https://docs.netapp.com/us-en/ontap/enable-snapshot-dir-access-task.html
Portworx Flashblade: uncheck Snapshot option https://docs.portworx.com/portworx-backup-on-prem/reference/restore-with-fb

Reply:
@kaovilai - Ach let us give it a lash

![image](https://github.com/user-attachments/assets/e24e3c8c-009a-4c6b-8045-2be6aa854ef6)

I think it looks ok... maybe nobody will notice